Content & Docs handbook update (#10094)
PostHog/posthog.com: contents/handbook/growth/marketing/index.md

showTitle: true
---

## How marketing works

Marketing at PostHog is a collaborative effort dispersed across three teams:

- **Content & Docs:** Responsible for content marketing, SEO, and maintaining documentation.

> - **Words & Pictures:** Responsible for product marketing, art work, paid ads, and email marketing.

Reviewer: ```suggestion
- **Words & Pictures:** Responsible for product marketing, artwork, paid ads, events, and email marketing.
```

> - **Website & Vibes:** Responsible for website design, community, and backend.

Author: @joethreepwood @corywatilo @charlescook-ph Added this overview to the marketing handbook to clarify top-level ownership of stuff.

 
- Don't pretend our customers are different from us – i.e. more gullible, more susceptible to marketing. We are an engineering-led team building products for other engineers. If you wouldn't like it, assume our customers wouldn't either. 

## Marketing vision 

### Things we want to be brilliant at
 
- **Word of mouth mindset:** By IPO, we want to build a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.

- **Speed:** We want to be highly reactive, low process, and reliant on other teams as little as possible to ship things. We want to get stuff wrong quickly, then iterate.

- **Helping PostHog users be successful:** Through our docs, tutorials, newsletter and beyond, we help PostHog users be more successful, both generally in their goals, but also more successful and powerful users of PostHog.

- **Content for our ideal customer:** We deliver genuinely useful insights about things those customers care about (can be purely product-related, but also general advice). We're pitching our content at ‘self-servers’.

> - **Documentation** We work with product teams to maintain update-to-date, high quality docs. We work with Website & Docs to ensure users can discover what they want to know. 

Reviewer 2: ```suggestion
- **Documentation** We work with product teams to maintain update-to-date, high quality docs. We also want to ensure users can discover what they want to know. 
```

 
### Things we want to be good at

> - **Social media:** Specifically Twitter, where we've seen good traction posting on James' personal account and the PostHog brand account. We do a minimal amount of posting on PostHog's LinkedIn account to keep it active, but it's not an important channel for us. We don't use any other social media channels. 

Reviewer 2: ```suggestion
- **Social media:** Specifically Twitter, where we've seen good traction posting on [James' personal account](https://x.com/james406) and the PostHog brand account. We do a minimal amount of posting on PostHog's LinkedIn account to keep it active, but it's not an important channel for us. We don't use any other social media channels. 
```

> - **Developer influencers:** A small but growing trend in the industry, most of these are very expensive and overwhelmingly focused on teaching people how to code (i.e. not our target audience). We tried this with an agency and didn't get traction, but have seen some early promising signs going to developers directly. 

Reviewer 2: ```suggestion
- **Developer influencers:** We sponsor creators like Theo and Fireship to drive awareness and signups to PostHog. Many of the influencers we sponsor don't work out, but the ones that work drive great results. 
```

 
- Don't pretend our customers are different from us – i.e. more gullible, more susceptible to marketing. We are an engineering-led team building products for other engineers. If you wouldn't like it, assume our customers wouldn't either. 

## Marketing vision 

### Things we want to be brilliant at
 

> - **Word of mouth mindset:** By IPO, we want to have built a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.

Reviewer: ```suggestion
- **Word of mouth mindset:** By IPO, we want to build a hugely successful company driven primarily by word of mouth, rather than paid ads or PR.
```

- **Speed:** We want to be highly reactive, low process, and reliant on other teams as little as possible to ship things. We want to get stuff wrong quickly, then iterate.

- **Helping PostHog users be successful:** Through our docs, tutorials, newsletter and beyond, we help PostHog users be more successful, both generally in their goals, but also more successful and powerful users of PostHog.

- **Content for our ideal customer:** We deliver genuinely useful insights about things those customers care about (can be purely product-related, but also general advice). We're pitching our content at ‘self-servers’.

- **Documentation** We work with product teams to maintain update-to-date, high quality docs. We work with Website & Docs to ensure users can discover what they want to know. 
 
### Things we want to be good at

- **Social media:** Specifically Twitter, where we've seen good traction posting on James' personal account and the PostHog brand account. We do a minimal amount of posting on PostHog's LinkedIn account to keep it active, but it's not an important channel for us. We don't use any other social media channels. 

> - **Paid ads:** An agency should keep handling this for us, but we shouldn’t get better at it internally. This includes out of home, should we ever decide to do this. We see paid ads as a hygiene factor. 

Reviewer: ```suggestion
```
